git_machete/client.py

```diff
@@ -91,8 +91,8 @@ def get_childless_branches(self) -> List[LocalBranchShortName]:
     def expect_in_managed_branches(self, branch: LocalBranchShortName) -> None:
         if branch not in self.managed_branches:
             raise MacheteException(
-                f"Branch `{branch}` not found in the tree of branch dependencies.\n"
-                f"Use `git machete add {branch}` or `git machete edit`")
+                f"Branch {bold(branch)} not found in the tree of branch dependencies.\n"
+                f"Use `git machete add {bold(branch)}` or `git machete edit`")
```

Comment:
Revert this like, `bold` inside of shell command like here isn't needed. I'd say that command formatting should take priority